samcli/hook_packages/terraform/hooks/prepare/hook.py

```diff
@@ -1341,7 +1345,7 @@ def _build_makerule_python_command(
     )
     return show_command_template.format(
         python_command_name=python_command_name,
-        terraform_built_artifacts_script_path=terraform_built_artifacts_script_path,
+        terraform_built_artifacts_script_path=str(terraform_built_artifacts_script_path).replace("\\", "/"),
         jpath_string=jpath_string.replace('"', '\\"'),
```

Comment:
passing the path in a unix format so it can work on all OSs, and also to work if the customer executes sam build in container.